code/modules/gear_presets/upp.dm

```diff
@@ -2637,6 +2637,9 @@
 /datum/equipment_preset/upp/synth/load_race(mob/living/carbon/human/new_human)
 	new_human.set_species(SYNTH_GEN_THREE)
 
+/datum/equipment_preset/upp/synth/load_skills(mob/living/carbon/human/new_human)
+	new_human.allow_gun_usage = FALSE
```

Comment:
This PR might want to change the name of the synth's role from `"UPP Combat Synthetic"` as well